pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -66,6 +66,10 @@ func getEnvTemplate(cr *v1.KieApp) v1.EnvTemplate {
 	if len(cr.Spec.Objects.Server.Env) == 0 {
 		cr.Spec.Objects.Server.Env = []corev1.EnvVar{{Name: "empty"}}
 	}
+	if reflect.DeepEqual(cr.Spec.RhpamRegistry, v1.KieAppRegistry{}) {
```

Comment:
Should be able to do "==" instead of DeepEqual

Reply:
just tried and it would look like this instead ... want me to change?
```golang
	var emptyReg v1.KieAppRegistry
	if cr.Spec.RhpamRegistry == emptyReg {
		cr.Spec.RhpamRegistry.Registry = shared.GetEnv("REGISTRY", constants.RhpamRegistry) // default to red hat registry
		cr.Spec.RhpamRegistry.Insecure = shared.GetBoolEnv("INSECURE")
	}
```

Reply:
Yes, please

Reply:
changing to this:
```golang
	if cr.Spec.RhpamRegistry == (v1.KieAppRegistry{}) {
		cr.Spec.RhpamRegistry.Registry = shared.GetEnv("REGISTRY", constants.RhpamRegistry) // default to red hat registry
		cr.Spec.RhpamRegistry.Insecure = shared.GetBoolEnv("INSECURE")
	}
```

Reply:
done